packages/react-core/src/components/Page/PageHeaderToolsGroup.tsx

```diff
@@ -11,11 +11,11 @@ export interface PageHeaderToolsGroupProps extends React.HTMLProps<HTMLDivElemen
   /** Visibility at various breakpoints. */
   visibility?: {
     default: 'hidden' | 'visible';
```

Comment:
This should be optional too, no?

Reply:
I figured the intent was that default is supposed to be enforced and exceptions specified, but now looking at how some of the other components work:

      <Flex direction={{}}>

Seems like default is generally optional. Let me update real quick...